recOrder/cli/compute_transfer_function.py

```diff
@@ -194,26 +192,30 @@ def compute_transfer_function_cli(input_data_path, config_path, output_path):
     # Write settings to metadata
     output_dataset.zattrs["settings"] = settings.dict()
 
-    echo_headline(f"Closing {output_path}\n")
+    echo_headline(f"Closing {output_dirpath}\n")
     output_dataset.close()
 
     echo_headline(
-        f"Recreate this transfer function with:\n$ recorder compute-tf {input_data_path} -c {config_path} -o {output_path}"
+        f"Recreate this transfer function with:\n$ recorder compute-tf {input_position_dirpaths} -c {config_filepath} -o {output_dirpath}"
     )
 
 
 @click.command()
-@click.help_option("-h", "--help")
-@input_data_path_argument()
-@config_path_option()
-@output_dataset_option(default="./transfer-function.zarr")
-def compute_tf(input_data_path, config_path, output_path):
+@input_position_dirpaths()
+@config_filepath()
+@output_dirpath()
+def compute_tf(
+    input_position_dirpaths: List[str],
+    config_filepath: str,
+    output_dirpath: int,
+):
     """
     Compute a transfer function using a dataset and configuration file.
 
     See /examples/ for example configuration files.
 
-    Example usage:\n
-    $ recorder compute-tf input.zarr/0/0/0 -c /examples/birefringence.yml -o transfer_function.zarr
+    >> recorder compute-tf -i ./input.zarr/0/0/0 -c ./examples/birefringence.yml -o ./transfer_function.zarr
     """
-    compute_transfer_function_cli(input_data_path, config_path, output_path)
+    compute_transfer_function_cli(
+        input_position_dirpaths[0], config_filepath, output_dirpath
```

Comment:
Need to document the assumption that all input paths store arrays of the same shape.

Reply:
Good call. I clarified in the help string for that parameter.

Reply:
Can you link to the line? The diff is too big...

Reply:
Sorry I couldn't find the direct link button. See below cli.parsing L34 see link below